sessionctx/variable/session.go

```diff
@@ -1279,6 +1279,8 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		} else {
 			s.StmtCtx.AppendWarning(errors.Errorf("cannot update %s when enabling dynamic configs", TiDBEnableCollectExecutionInfo))
 		}
+	case TiDBFreeOSMemoryThreshold:
+		atomic.StoreUint64(&config.GetGlobalConfig().FreeOSMemoryThreshold, uint64(tidbOptInt64(val, 0)))
```

Comment:
Do we need a lower limit?

Reply:
I think no need for this, such as `mem-quota-query` also has no lower/higher limit.